src/main/java/com/conveyal/taui/controllers/SinglePointAnalysisController.java

```diff
@@ -1,88 +0,0 @@
-package com.conveyal.taui.controllers;
-
-import com.conveyal.r5.analyst.cluster.TravelTimeSurfaceTask;
-import com.conveyal.r5.common.JsonUtilities;
-import com.conveyal.taui.AnalysisServerConfig;
-import com.conveyal.taui.models.AnalysisRequest;
-import com.conveyal.taui.models.Project;
-import com.conveyal.taui.persistence.Persistence;
-import com.conveyal.taui.util.HttpUtil;
-import com.google.common.io.ByteStreams;
-import org.apache.http.client.methods.CloseableHttpResponse;
-import org.apache.http.client.methods.HttpPost;
-import org.apache.http.entity.ContentType;
-import org.apache.http.entity.StringEntity;
-import org.apache.http.util.EntityUtils;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import spark.Request;
-import spark.Response;
-
-import java.io.BufferedInputStream;
-import java.io.ByteArrayOutputStream;
-import java.io.IOException;
-import java.io.InputStream;
-
-import static spark.Spark.post;
-
-/**
- * Handles talking to the broker.
- */
-public class SinglePointAnalysisController {
-    private static final Logger LOG = LoggerFactory.getLogger(SinglePointAnalysisController.class);
-    private static final String BROKER_ENQUEUE_SINGLE_URL = AnalysisServerConfig.brokerUrl + "/enqueue/single";
-
-    public static byte[] analysis (Request req, Response res) throws IOException {
-        // we already know the user is authenticated, and we need not check if they have access to the graphs etc,
-        // as they're all coded with UUIDs which contain significantly more entropy than any human's account password.
-        final String accessGroup = req.attribute("accessGroup");
-        final String email = req.attribute("email");
-
-        AnalysisRequest analysisRequest = JsonUtilities.objectMapper.readValue(req.body(), AnalysisRequest.class);
-        Project project = Persistence.projects.findByIdIfPermitted(analysisRequest.projectId, accessGroup);
-        TravelTimeSurfaceTask task = (TravelTimeSurfaceTask) analysisRequest.populateTask(new TravelTimeSurfaceTask(), project);
-
-        if (req.headers("Accept").equals("image/tiff")) {
-            task.setFormat(TravelTimeSurfaceTask.Format.GEOTIFF);
```

Comment:
Is this functionality being maintained?

Reply:
The GEOTIFF functionality is still provided in R5's com.conveyal.r5.analyst.cluster.AnalystWorker#handleOneRequest (but with some hard-coded headers that might cause problems in `com.conveyal.r5.analyst.cluster.AnalysisWorkerController`). However the HTTP proxy method com.conveyal.taui.controllers.WorkerController#singlePoint was not converting the `image/tiff` header into TravelTimeSurfaceTask.Format.GEOTIFF on the TravelTimeSurfaceTask. Will fix this.

Reply:
I tested my fix locally and am able to produce Geotiffs.